README.md

```diff
@@ -15,7 +15,8 @@ To use Toolkit in your project:
 * **[Bookmarks](https://developers.arcgis.com/swift/toolkit-api-reference/documentation/arcgistoolkit/bookmarks)** - Shows bookmarks, from a map, scene, or a list.
 * **[Compass](https://developers.arcgis.com/swift/toolkit-api-reference/documentation/arcgistoolkit/compass)** - Shows a compass direction when the map is rotated. Auto-hides when the map points north.
 * **[FloatingPanel](https://developers.arcgis.com/swift/toolkit-api-reference/documentation/arcgistoolkit/floatingpanel)** - Allows display of view-related content in a "bottom sheet". 
-* **[FloorFilter](https://developers.arcgis.com/swift/toolkit-api-reference/documentation/arcgistoolkit/floorfilter)** - Allows filtering of floor plan data in a geo view by a site, a building in the site, or a floor in the building. 
+* **[FloorFilter](https://developers.arcgis.com/swift/toolkit-api-reference/documentation/arcgistoolkit/floorfilter)** - Allows filtering of floor plan data in a geo view by a site, a building in the site, or a floor in the building.
+* **[JobManager](https://developers.arcgis.com/swift/toolkit-api-reference/documentation/arcgistoolkit/jobmanager)** - Manages saving and loading jobs so that they can continue to run if the app is backgrounded or even terminated.
```

Comment:
> or even terminated

Does this conflict with the following statement from `JobManager`'s doc?

```swift
/// Also in iOS if the user of an app removes the app from the app switcher (swiping up) then any
/// background downloads in progress are canceled by the operating system. This behavior is intended
/// by iOS and any jobs that were in the process of downloading results will cease downloading.
```

I started a job and then killed the app via the switcher and it didn't finish after ~4 minutes.

Or, is the intent to communicate that the job can be resumed from where it was once the app is relaunched?

Reply:
It only works if you do not terminate the app via app switcher.  The doc above doesn't go into that detail because it is a lot to explain. The doc below goes into the details that it doesn't work if the user kills it via app switcher.  Apple (iOS) will not give background time to an app that the user manually removes from the app switcher.  There is no good way around that, it's by design.  If that's not clear enough, I'm open to suggestions.

Reply:
Understood. Here would be my suggestion but okay to keep as is if you prefer:

```suggestion
* **[JobManager](https://developers.arcgis.com/swift/toolkit-api-reference/documentation/arcgistoolkit/jobmanager)** - Manages saving and loading jobs so that they can continue to run if the app is backgrounded or resumed after the app is terminated and relaunched.
```

Reply:
Ok clarified both sections of doc. See what you think please.